collectors/python.d.plugin/varnish/metadata.yaml

```diff
@@ -36,14 +44,48 @@ setup:
     list: []
   configuration:
     file:
-      name: ''
+      name: python.d/varnish.conf
       description: ''
     options:
-      description: ''
+      description: |
+        There are 2 sections:
+
+        * Global variables
+        * One or more JOBS that can define multiple different instances to monitor.
+
+        The following options can be defined globally: priority, penalty, autodetection_retry, update_every, but can also be defined per JOB to override the global values.
+
+        Additionally, the following table contains all the options that can be configured inside a JOB definition.
+
+        Every configuration JOB starts with a `job_name` value which will appear in the dashboard, unless a `name` parameter is specified.
       folding:
-        title: ''
+        title: Config options
         enabled: true
-      list: []
+      list:
+        - name: instance_name
+          description: the name of the varnishd instance to get logs from. If not specified, the host name is used.
+          default_value: '<host name>'
+          required: true
+        - name: update_every
+          description: Sets the default data collection frequency.
+          default_value: 10
+          required: false
+        - name: priority
+          description: Controls the order of charts at the netdata dashboard.
+          default_value: 60000
+          required: false
+        - name: autodetection_retry
+          description: Sets the job re-check interval in seconds.
+          default_value: 0
+          required: false
+        - name: penalty
+          description: Indicates whether to apply penalty to update_every in case of failures.
+          default_value: yes
+          required: false
+        - name: name
+          description: Job name. This value will overwrite the `job_name` value. JOBS with the same name are mutually exclusive. Only one of them will be allowed running at any time. This allows autodetection to try several alternatives and pick the one that works.
+          default_value: ''
+          required: false
     examples:
```

Comment:
Not providing examples here is kind of valid, there is not much the user can configure, only the instance_name. @ilyam8 what do you think?

Reply:
added basic example here https://github.com/netdata/netdata/pull/15528/commits/6bade502755611a58d8721a68f717933cf9377bf